data/core/units/elves/Marksman.cfg

```diff
@@ -16,7 +16,7 @@
     advances_to=Elvish Sharpshooter
     cost=37
     usage=archer
-    description= _ "Elves are wondrously keen-sighted, a gift which contributes to their mastery of the bow. An elf practiced at archery can hit targets that a man couldn’t even see, and can nock a second arrow almost as soon as the first is let fly. Realizing this prodigious skill does come with one cost, that being a lack of practice with the sword."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}
+    description= _ "Years of hunting in the deep forests combined with magically amplified senses makes the elves incredibly keen-sighted, a gift which contributes greatly to their mastery of the bow. An elf practiced at archery is capable of hitting any target, no matter how quickly it is moving or how far it is. Of course, training the body to keep up with the eyes is no easy feat, and realizing this prodigious skill does come with the cost of weakness in close quarters."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}
```

Comment:
No, drop the magical enhancement and go back to natural selection. Sure, it you or me, their senses (at least four of the five: sight, smell, hearing and touch) may seem unnatural and magical.

Reply:
Check out the elf race description, it says they're magical beings (at least somewhat).

Reply:
Maybe it's just me, but when I read "magically amplified sense" it sounds to me like there's an agent that amplifies the elves' senses magically.

I prefer "can hit targets that a man couldn’t even see" to "no matter how quickly it is moving or how far it is". The former is more vivid, and positive; the latter basically leaves it to the reader to imagine just _how_ quick a target can an Elvish Marksman hit.

Reply:
To your second point, that is one of the phrases I thought pretty blatantly showed elf-bias. I can find a way to reword my description so it's more specific.